src/test/ui/lint/dead-code/type-in-foreign.rs

```diff
@@ -0,0 +1,11 @@
+// check-pass
+
+#[repr(C)]
+struct Type(u8);
+
+extern "C" {
+    #[allow(dead_code)]
```

Comment:
Shouldn't this instead `warn(dead_code)` and check the error is correct?

Reply:
I should add a comment: the test must not warn on `Type` line 4.

Reply:
can you add `#[deny(dead_code)]` to this test?